WordPress/Tests/Arrays/ArrayDeclarationSpacingUnitTest.inc

```diff
@@ -2,7 +2,23 @@
 
 $good = array( 'value1', 'value2' ); // Ok.
 
-$query_vars = array('post_type' => 'food'); // Bad, no spaces after opening and before closing parenthesis.
+$query_vars = array('food'); // Bad, no spaces after opening and before closing parenthesis.
 
 // Test for fixing of extra whitespace.
 $test = array(   1, 2     );
+
+$bad = array( 'key' => 'value' ); // Bad, each value of an associative array should start on a new line.
```

Comment:
The handbook doesn't say anywhere that multi-value arrays should be multi-line (though I totally agree that that would be good practice).
It only has the bit we both quoted and that bit doesn't specify that it doesn't apply to single-value associative arrays...